fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -499,26 +503,28 @@ public FHIRRestOperationResponse doDelete(String type, String id, String searchQ
             // resource to be deleted. Otherwise, we'll use the id value to identify the resource
             // to be deleted.
             Resource resourceToDelete = null;
+            Bundle responseBundle = null;
+
             if (searchQueryString != null) {
+                int searchPageSize = FHIRConfigHelper.getIntProperty(FHIRConfiguration.PROPERTY_CONDITIONAL_DELETE_MAX_NUMBER, FHIRConstants.FHIR_CONDITIONAL_DELETE_MAX_NUMBER_DEFAULT);
```

Comment:
because you use FHIRConfigHelper to look it up, I think this means it *is* client-specific

Reply:
yes, I agree. just was a little bit confused about the document.